editor.planx.uk/src/pages/FlowEditor/components/forms/FormModal.tsx

```diff
@@ -46,6 +46,7 @@ const NodeTypeSelect: React.FC<{
       <optgroup label="Question">
         <option value={TYPES.Statement}>Question</option>
         <option value={TYPES.Checklist}>Checklist</option>
+        {/* <option value={TYPES.NextSteps}>Next steps</option> */}
```

Comment:
This grouping makes sense to me too :heavy_plus_sign:

As long as the Public display shows "Under development", I'd actually be in favor of this _not_ being commented out in this list so we can easily test/preview on the pizza! And that should be enough of a signal to Editors to not use it yet

Reply:
Thanks, I've updated so this is unhidden, also updated the description with a pizza preview link for the component.